sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -235,7 +232,7 @@ export class ECertFileHandler extends ESDCFileHandler {
       educationalEndDate: offering.studyEndDate,
       federalInstitutionCode: offering.institutionLocation.institutionCode,
       weeksOfStudy: application.currentAssessment.assessmentData.weeks,
-      fieldOfStudy,
+      fieldOfStudy: offering.educationProgram.fieldOfStudyCode,
```

Comment:
fieldOfStudyCode is not added to the select which is used by this method.

![image](https://user-images.githubusercontent.com/54600590/184732957-fcb13713-a90a-45ce-8bbb-2c6aeac155c5.png)